api/secrets/api.go

```diff
@@ -31,21 +31,17 @@ type Service interface {
 
 // Secret hold sensitive data
 type Secret struct {
-	ID          string            `json:"ID"`
-	Name        string            `json:"Name"`
-	Labels      map[string]string `json:"Labels"`
-	Description string            `json:"Description"`
-	username    string
-	password    string
+	ID      string            `json:"ID"`
+	Name    string            `json:"Name"`
+	Tags    map[string]string `json:"Tags"`
```

Comment:
Why was this changed from Labels to Tags? Are these Azure or ECS-specific?

If these are the equivalent to labels, we could use `--label`, and translate them to `--tag` for the API call.
Alternatively (if they're separate from "user" labels), we could consider having a prefix (`--label com.azure.tag=foo`) 

(mostly curious) 😅

Reply:
It's ECS specific :sweat_smile: We tag most of AWS resources we create with compose and I thought it is easier to follow if I renamed Labels to Tags.

Reply:
If the plan here is to use AWS tags to implement docker labels, then should better avoid to expose AWS terminology. 

If we go this way, I wonder how TaksDefinition's labels should be set, as it has both `Tags` and  `DockerLabels`. Maybe configured labels should be applied twice there.

Reply:
Ok, I will change back to Labels :sweat:

Reply:
Thanks! (and sorry for being a PITA!). We should definitely look if there's feature-disparity between the backends, and (where possible) re-use existing terminology to keep the UX consistent. At the same time, if we find things that will affect the usability too much, we should have a look if we can find some common (platform/backend-agnostic) way to address these